cmd/skycoin/skycoin.go

```diff
@@ -434,6 +434,56 @@ func catchDebug() {
 	}
 }
 
+func createGUI(c *Config, d *daemon.Daemon, wg sync.WaitGroup) *gui.Server {
```

Comment:
sync.WaitGroup cannot be used like this


>A WaitGroup waits for a collection of goroutines to finish. The main goroutine calls Add to set the number of goroutines to wait for. Then each of the goroutines runs and calls Done when finished. At the same time, Wait can be used to block until all goroutines have finished.
>A WaitGroup must not be copied after first use.

https://golang.org/pkg/sync/#WaitGroup

You must use it from the same function it is created